src/puyapy/awst_build/arc4_decorators.py

```diff
@@ -231,6 +231,8 @@ def visit_name_expr(self, o: mypy.nodes.NameExpr) -> object:
         elif self.arg_name == _CLIENT_DEFAULTS:
             if isinstance(o.node, mypy.nodes.Decorator):
                 return o.name  # assume abimethod
+            if o.fullname in ["builtins.True", "builtins.False"]:
```

Comment:
```suggestion
            if o.fullname in ("builtins.True", "builtins.False"):
```

Usually we'd use a `tuple` literal here

Reply:
I've updated and pushed it.